packages/config/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moonbeam-network/xcm-config",
-  "version": "1.0.3",
+  "version": "0.0.1-dev.47",
```

Comment:
**Note:** `dev` versioning had to be manually modified for it to work. Please ignore the changes in the `package.json` files, as I'll revert them before merging